runtime/adapters/src/lib.rs

```diff
@@ -491,6 +495,13 @@ where
 		)?;
 		Ok(vec![staking_used, referrals_used])
 	}
+
+	fn consume_protocol_fee(fee_account: AccountId, amount: Balance) -> Result<(Balance, AccountId), Self::Error> {
+		//TODO: here in future, we will change this to buyback HDX with the lrna amount
```

Comment:
we could maybe add this link here in the comment: https://github.com/galacticcouncil/hydration-node/issues/969